src/backend/variables/builtin/twitch/chat/message/chat-user-color.ts

```diff
@@ -0,0 +1,59 @@
+import { ReplaceVariable } from "../../../../../../types/variables";
+import { EffectTrigger } from "../../../../../../shared/effect-constants";
+import { OutputDataType, VariableCategory } from "../../../../../../shared/variable-constants";
+import accountAccess from "../../../../../common/account-access";
+import viewerDatabase from "../../../../../viewers/viewer-database";
+
+const twitchApi = require("../../../../../twitch-api/api");
+
+const model : ReplaceVariable = {
+    definition: {
+        handle: "chatUserColor",
+        description: "Outputs the chatters display color from a command or event.",
+        examples: [
+            {
+                usage: "chatUserColor[$target]",
+                description: "When in a command, gets the users color for the target user."
+            },
+            {
+                usage: "chatUserColor[$user]",
+                description: "Gets the color for associated user (Ie who triggered command, pressed button, etc)."
+            }
+        ],
+        categories: [VariableCategory.COMMON, VariableCategory.TRIGGER, VariableCategory.USER],
+        possibleDataOutput: [OutputDataType.TEXT]
+    },
+    evaluator: async (trigger, username: string) => {
+        try {
+            let chatColor = "";
+            if (username !== null) {
```

Comment:
Is there a specific need to only check `null` but not `undefined` here (`!== null`)? If not, using `!= null` like we do below should be used